examples/osgi-test-example-mvn/org.osgi.test.example.player.impl/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -165,7 +165,6 @@ org.eclipse.jdt.core.formatter.alignment_for_arguments_in_explicit_constructor_c
 org.eclipse.jdt.core.formatter.alignment_for_arguments_in_method_invocation=16
 org.eclipse.jdt.core.formatter.alignment_for_arguments_in_qualified_allocation_expression=16
 org.eclipse.jdt.core.formatter.alignment_for_assignment=0
-org.eclipse.jdt.core.formatter.alignment_for_binary_expression=16
 org.eclipse.jdt.core.formatter.alignment_for_bitwise_operator=16
```

Comment:
This is in the osgi-test-example-mvn project.

Reply:
used a wrong git add filter